broken package bfd

```
make[1]: Entering directory `/home/mxe/mxe'
uname -a
Linux sd-68075 3.2.0-4-amd64 #1 SMP Debian 3.2.63-2+deb7u1 x86_64 GNU/Linux
git show-branch --list --reflog=1
  [refs/heads/master@{0}] (2 hours ago) pull -u: Fast-forward
lsb_release -a 2>/dev/null || sw_vers 2>/dev/null || true
Distributor ID:	Debian
Description:	Debian GNU/Linux 7.8 (wheezy)
Release:	7.8
Codename:	wheezy
rm -rf   '/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static'
mkdir -p '/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static'
cd '/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static' &&    tar xjf '/home/mxe/mxe/pkg/binutils-2.25.tar.bz2'
cd '/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25'
cd '/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/bfd' && ./configure --host='i686-w64-mingw32.static' --target='i686-w64-mingw32.static' --prefix='/home/mxe/mxe/usr/i686-w64-mingw32.static' --enable-install-libbfd --disable-shared
configure: WARNING: If you wanted to set the --build type, don't use --host.
    If a cross compiler is detected then cross compile mode will be used.
configure: loading site script /home/mxe/mxe/usr/i686-w64-mingw32.static/share/config.site
checking build system type... (cached) x86_64-unknown-linux-gnu
checking host system type... i686-w64-mingw32.static
checking target system type... i686-w64-mingw32.static
checking for i686-w64-mingw32.static-gcc... i686-w64-mingw32.static-gcc
checking for C compiler default output file name... a.exe
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables... .exe
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether i686-w64-mingw32.static-gcc accepts -g... yes
checking for i686-w64-mingw32.static-gcc option to accept ISO C89... none needed
checking for library containing strerror... none required
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking for style of include used by make... GNU
checking dependency style of i686-w64-mingw32.static-gcc... gcc3
checking for i686-w64-mingw32.static-ar... i686-w64-mingw32.static-ar
checking for i686-w64-mingw32.static-ranlib... i686-w64-mingw32.static-ranlib
checking for i686-w64-mingw32.static-gcc... (cached) i686-w64-mingw32.static-gcc
checking whether we are using the GNU C compiler... (cached) yes
checking whether i686-w64-mingw32.static-gcc accepts -g... (cached) yes
checking for i686-w64-mingw32.static-gcc option to accept ISO C89... (cached) none needed
checking how to run the C preprocessor... i686-w64-mingw32.static-gcc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking how to print strings... printf
checking for a sed that does not truncate output... /bin/sed
checking for fgrep... /bin/grep -F
checking for ld used by i686-w64-mingw32.static-gcc... /home/mxe/mxe/usr/bin/i686-w64-mingw32.static-ld
checking if the linker (/home/mxe/mxe/usr/bin/i686-w64-mingw32.static-ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /home/mxe/mxe/usr/bin/i686-w64-mingw32.static-nm -B
checking the name lister (/home/mxe/mxe/usr/bin/i686-w64-mingw32.static-nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 3458764513820540925
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking for /home/mxe/mxe/usr/bin/i686-w64-mingw32.static-ld option to reload object files... -r
checking for i686-w64-mingw32.static-objdump... i686-w64-mingw32.static-objdump
checking how to recognize dependent libraries... file_magic ^x86 archive import|^x86 DLL
checking for i686-w64-mingw32.static-ar... (cached) i686-w64-mingw32.static-ar
checking for i686-w64-mingw32.static-strip... i686-w64-mingw32.static-strip
checking for i686-w64-mingw32.static-ranlib... (cached) i686-w64-mingw32.static-ranlib
checking command to parse /home/mxe/mxe/usr/bin/i686-w64-mingw32.static-nm -B output from i686-w64-mingw32.static-gcc object... ok
checking for dlfcn.h... yes
checking for objdir... .libs
checking if i686-w64-mingw32.static-gcc supports -fno-rtti -fno-exceptions... no
checking for i686-w64-mingw32.static-gcc option to produce PIC... -DDLL_EXPORT -DPIC
checking if i686-w64-mingw32.static-gcc PIC flag -DDLL_EXPORT -DPIC works... yes
checking if i686-w64-mingw32.static-gcc static flag -static works... yes
checking if i686-w64-mingw32.static-gcc supports -c -o file.o... yes
checking if i686-w64-mingw32.static-gcc supports -c -o file.o... (cached) yes
checking whether the i686-w64-mingw32.static-gcc linker (/home/mxe/mxe/usr/bin/i686-w64-mingw32.static-ld) supports shared libraries... yes
checking dynamic linker characteristics... Win32 ld.exe
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... no
checking whether to build static libraries... yes
checking for dlfcn.h... (cached) yes
checking for windows.h... yes
checking for library containing dlopen... -ldl
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... 64
Setting warning flags = -W -Wall -Wstrict-prototypes -Wmissing-prototypes -Wshadow -Wno-format
checking whether to enable maintainer-specific portions of Makefiles... no
checking whether to install libbfd... yes
checking whether NLS is requested... no
checking whether NLS is requested... yes
checking for msgfmt... /usr/bin/msgfmt
checking for gmsgfmt... /usr/bin/msgfmt
checking for xgettext... /usr/bin/xgettext
checking for msgmerge... /usr/bin/msgmerge
checking size of long long... 8
checking size of void *... 4
checking size of long... 4
checking alloca.h usability... no
checking alloca.h presence... no
checking for alloca.h... no
checking stddef.h usability... yes
checking stddef.h presence... yes
checking for stddef.h... yes
checking for string.h... (cached) yes
checking for strings.h... (cached) yes
checking for stdlib.h... (cached) yes
checking time.h usability... yes
checking time.h presence... yes
checking for time.h... yes
checking for unistd.h... (cached) yes
checking wchar.h usability... yes
checking wchar.h presence... yes
checking for wchar.h... yes
checking fcntl.h usability... yes
checking fcntl.h presence... yes
checking for fcntl.h... yes
checking sys/file.h usability... yes
checking sys/file.h presence... yes
checking for sys/file.h... yes
checking sys/time.h usability... yes
checking sys/time.h presence... yes
checking for sys/time.h... yes
checking for sys/stat.h... (cached) yes
checking sys/resource.h usability... no
checking sys/resource.h presence... no
checking for sys/resource.h... no
looking for a compliant stdint.h in stdint.h, checking for uintmax_t... yes
checking for uintptr_t... yes
checking for int_least32_t... yes
checking for int_fast32_t... yes
checking for uint64_t... yes
checking what to include in bfd_stdint.h... stdint.h (already complete)
checking whether time.h and sys/time.h may both be included... yes
checking for dirent.h that defines DIR... yes
checking for library containing opendir... none required
checking whether string.h and strings.h may both be included... yes
checking for fcntl... no
checking for getpagesize... yes
checking for setitimer... no
checking for sysconf... no
checking for fdopen... yes
checking for getuid... no
checking for getgid... no
checking for fileno... yes
checking for strtoull... yes
checking for getrlimit... no
checking whether basename is declared... no
checking whether ftello is declared... yes
checking whether ftello64 is declared... yes
checking whether fseeko is declared... yes
checking whether fseeko64 is declared... yes
checking whether ffs is declared... no
checking whether free is declared... yes
checking whether getenv is declared... yes
checking whether malloc is declared... yes
checking whether realloc is declared... yes
checking whether stpcpy is declared... no
checking whether strstr is declared... yes
checking whether snprintf is declared... yes
checking whether vsnprintf is declared... yes
checking whether strnlen is declared... yes
checking for library containing zlibVersion... -lz
checking zlib.h usability... yes
checking zlib.h presence... yes
checking for zlib.h... yes
checking sys/procfs.h usability... no
checking sys/procfs.h presence... no
checking for sys/procfs.h... no
checking linker --as-needed support... yes
checking for cos in -lm... yes
checking for ftello... yes
checking for ftello64... yes
checking for fseeko... yes
checking for fseeko64... yes
checking for fopen64... yes
checking size of off_t... 8
checking file_ptr type... BFD_HOST_64_BIT
checking for stdlib.h... (cached) yes
checking for unistd.h... (cached) yes
checking for getpagesize... (cached) yes
checking for working mmap... no
checking for madvise... no
checking for mprotect... yes
configure: creating ./config.status
config.status: creating Makefile
config.status: creating doc/Makefile
config.status: creating bfd-in3.h
config.status: creating po/Makefile.in
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
config.status: executing default-1 commands
config.status: executing bfd_stdint.h commands
config.status: executing default commands
make -C '/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/bfd' -j '1'
make[2]: Entering directory `/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/bfd'
creating bfdver.h
rm -f elf32-target.h
sed -e s/NN/32/g < ./elfxx-target.h > elf32-target.new
mv -f elf32-target.new elf32-target.h
rm -f elf64-target.h
sed -e s/NN/64/g < ./elfxx-target.h > elf64-target.new
mv -f elf64-target.new elf64-target.h
rm -f targmatch.h
sed -f ./targmatch.sed < ./config.bfd > targmatch.new
mv -f targmatch.new targmatch.h
Making info in doc
make[3]: Entering directory `/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/bfd/doc'
i686-w64-mingw32.static-gcc -o chw$$.exe  \
	    -I.. -I./.. -I./../../include -I./../../intl -I../../intl ./chew.c; \
	/bin/bash ./../../move-if-change \
	  chw$$.exe chew.exe; \
	touch chew.stamp
./chew.exe -f ./doc.str < ../aoutx.h >aoutx.tmp
/bin/bash ./../../move-if-change aoutx.tmp aoutx.texi
touch aoutx.stamp
./chew.exe -f ./doc.str < ../archive.c >archive.tmp
/bin/bash ./../../move-if-change archive.tmp archive.texi
touch archive.stamp
./chew.exe -f ./doc.str < ../archures.c >archures.tmp
/bin/bash ./../../move-if-change archures.tmp archures.texi
touch archures.stamp
./chew.exe -f ./doc.str < ../bfd.c >bfd.tmp
/bin/bash ./../../move-if-change bfd.tmp bfdt.texi
touch bfdt.stamp
./chew.exe -f ./doc.str < ../cache.c >cache.tmp
/bin/bash ./../../move-if-change cache.tmp cache.texi
touch cache.stamp
./chew.exe -f ./doc.str < ../coffcode.h >coffcode.tmp
/bin/bash ./../../move-if-change coffcode.tmp coffcode.texi
touch coffcode.stamp
./chew.exe -f ./doc.str < ../corefile.c >core.tmp
/bin/bash ./../../move-if-change core.tmp core.texi
touch core.stamp
./chew.exe -f ./doc.str < ../elf.c >elf.tmp
/bin/bash ./../../move-if-change elf.tmp elf.texi
touch elf.stamp
./chew.exe -f ./doc.str < ../elfcode.h >elfcode.tmp
/bin/bash ./../../move-if-change elfcode.tmp elfcode.texi
touch elfcode.stamp
./chew.exe -f ./doc.str < ../format.c >format.tmp
/bin/bash ./../../move-if-change format.tmp format.texi
touch format.stamp
./chew.exe -f ./doc.str < ../libbfd.c >libbfd.tmp
/bin/bash ./../../move-if-change libbfd.tmp libbfd.texi
touch libbfd.stamp
./chew.exe -f ./doc.str < ../bfdwin.c >bfdwin.tmp
/bin/bash ./../../move-if-change bfdwin.tmp bfdwin.texi
touch bfdwin.stamp
./chew.exe -f ./doc.str < ../bfdio.c >bfdio.tmp
/bin/bash ./../../move-if-change bfdio.tmp bfdio.texi
touch bfdio.stamp
./chew.exe -f ./doc.str < ../opncls.c >opncls.tmp
/bin/bash ./../../move-if-change opncls.tmp opncls.texi
touch opncls.stamp
./chew.exe -f ./doc.str < ../reloc.c >reloc.tmp
/bin/bash ./../../move-if-change reloc.tmp reloc.texi
touch reloc.stamp
./chew.exe -f ./doc.str < ../section.c >section.tmp
/bin/bash ./../../move-if-change section.tmp section.texi
touch section.stamp
./chew.exe -f ./doc.str < ../syms.c >syms.tmp
/bin/bash ./../../move-if-change syms.tmp syms.texi
touch syms.stamp
./chew.exe -f ./doc.str < ../targets.c >targets.tmp
/bin/bash ./../../move-if-change targets.tmp targets.texi
touch targets.stamp
./chew.exe -f ./doc.str < ../init.c >init.tmp
/bin/bash ./../../move-if-change init.tmp init.texi
touch init.stamp
./chew.exe -f ./doc.str < ../hash.c >hash.tmp
/bin/bash ./../../move-if-change hash.tmp hash.texi
touch hash.stamp
./chew.exe -f ./doc.str < ../linker.c >linker.tmp
/bin/bash ./../../move-if-change linker.tmp linker.texi
touch linker.stamp
./chew.exe -f ./doc.str < ../mmo.c >mmo.tmp
/bin/bash ./../../move-if-change mmo.tmp mmo.texi
touch mmo.stamp
restore=: && backupdir=".am$$" && \
	rm -rf $backupdir && mkdir $backupdir && \
	if (/bin/bash /home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/missing --run makeinfo --version) >/dev/null 2>&1; then \
	  for f in bfd.info bfd.info-[0-9] bfd.info-[0-9][0-9] bfd.i[0-9] bfd.i[0-9][0-9]; do \
	    if test -f $f; then mv $f $backupdir; restore=mv; else :; fi; \
	  done; \
	else :; fi && \
	if /bin/bash /home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/missing --run makeinfo   -I . \
	 -o bfd.info `test -f 'bfd.texinfo' || echo './'`bfd.texinfo; \
	then \
	  rc=0; \
	else \
	  rc=$?; \
	  $restore $backupdir/* `echo "./bfd.info" | sed 's|[^/]*$||'`; \
	fi; \
	rm -rf $backupdir; exit $rc
/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/bfd/doc//mmo.texi:39: Cross reference to nonexistent node `mmo
 section mapping' (perhaps incorrect sectioning?).
/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/bfd/doc//hash.texi:45: Cross reference to nonexistent node `Deriving
 a New Hash Table Type' (perhaps incorrect sectioning?).
makeinfo: Removing output file `bfd.info' due to errors; use --force to preserve.
make[3]: *** [bfd.info] Error 1
make[3]: Leaving directory `/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/bfd/doc'
make[2]: *** [info-recursive] Error 1
make[2]: Leaving directory `/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/bfd'
make[1]: *** [build-only-bfd_i686-w64-mingw32.static] Error 2
make[1]: Leaving directory `/home/mxe/mxe'

real	0m15.676s
user	0m9.577s
sys	0m3.084s
```

There is `^M` after `mmo` and `Deriving` in the log file:

/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/bfd/doc//mmo.texi:39: Cross reference to nonexistent node 'mmo**^M** section mapping' (perhaps incorrect sectioning?).
/home/mxe/mxe/tmp-bfd-i686-w64-mingw32.static/binutils-2.25/bfd/doc//hash.texi:45: Cross reference to nonexistent node 'Deriving**^M** a New Hash Table Type' (perhaps incorrect sectioning?).

--- a/src/bfd-1-disable-subdir-doc.patch
+++ b/src/bfd-1-disable-subdir-doc.patch
@@ -1,0 +1,42 @@
+This file is part of MXE.
+See index.html for further information.
+
+From e64e4a2b049ff472d6400d9650e295c1ebd6294b Mon Sep 17 00:00:00 2001
+From: Boris Nagaev <bnagaev@gmail.com>
+Date: Wed, 12 Aug 2015 01:28:28 +0200
+Subject: [PATCH] bfd: disable subdir doc
+
+---
+ bfd/Makefile.am |    2 +-
+ bfd/Makefile.in |    2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/bfd/Makefile.am b/bfd/Makefile.am
+index 6f64f06..54f8747 100644
+--- a/bfd/Makefile.am
++++ b/bfd/Makefile.am
+@@ -23,7 +23,7 @@ ACLOCAL_AMFLAGS = -I . -I .. -I ../config
+ INCDIR = $(srcdir)/../include
+ CSEARCH = -I. -I$(srcdir) -I$(INCDIR)
+ 
+-SUBDIRS = doc po
++SUBDIRS = po
+ 
+ bfddocdir = doc
+ 
+diff --git a/bfd/Makefile.in b/bfd/Makefile.in
+index 2c385d5..2f2a552 100644
+--- a/bfd/Makefile.in
++++ b/bfd/Makefile.in
+@@ -339,7 +339,7 @@ AUTOMAKE_OPTIONS = 1.11 no-dist foreign
+ ACLOCAL_AMFLAGS = -I . -I .. -I ../config
+ INCDIR = $(srcdir)/../include
+ CSEARCH = -I. -I$(srcdir) -I$(INCDIR)
+-SUBDIRS = doc po
++SUBDIRS = po
+ bfddocdir = doc
+ libbfd_la_LDFLAGS = $(am__append_1) -release `cat libtool-soversion` \
+ 	@SHARED_LDFLAGS@ $(am__empty)
+-- 
+1.7.10.4
+

